frontend/utilities/helpers.ts

```diff
@@ -661,15 +661,20 @@ const DISK_ENCRYPTION_MESSAGES = {
   },
 };
 
-export const humanHostDiskEncryptionEnabled = (
-  platform?: string,
-  isDiskEncrypted = false
-): string => {
-  if (platform !== "windows" && platform !== "darwin") {
+export const getHostDiskEncryptionTooltipMessage = (
+  platform: "darwin" | "windows" | "chrome", // TODO: improve this type
```

Comment:
No need to make changes on this PR, but here's one possible approach using utility [types](https://www.typescriptlang.org/docs/handbook/utility-types.html#extracttype-union):

```ts
platform: Extract<typeof SUPPORTED_PLATFORMS[number], "darwin" | "chrome" | "windows">
```